vortex-serde/src/layouts/footer.rs

```diff
@@ -25,14 +26,20 @@ impl Footer {
         (self.schema_offset - self.leftovers_offset) as usize
     }
 
-    pub fn layout(&self) -> VortexResult<Layout> {
+    pub fn layout(
```

Comment:
maybe `prune_layout` or something like that?

Reply:
I think this is still something different, we are not pruning just reading with applied filters